docs/3.0-migration-guide.d/remove-enable-weak-ciphersuites.md

```diff
@@ -0,0 +1,9 @@
+Remove the configuration to enable weak ciphersuites in SSL / TLS
+-----------------------------------------------------------------
+
+This does not affect users who use the default `config.h`, as this option was
+already off by default.
+
+Please switch to any of the modern, recommended ciphersuites (based on
+AES-GCM, AES-CCM or ChachaPoly for example) and if your peer doesn't support
+any, encourage them to upgrade their software.
```

Comment:
This is accurate for users of ciphersuites with weak crypto such as DES or ARC4. It isn't accurate for users of ciphersuites without encryption. Those are now solely gated by `MBEDTLS_CIPHER_NULL_CIPHER`.

Reply:
Thus what do you suggest? I just remove this ?

Reply:
No, it's correct, but incomplete. Please qualify that sentence with something like “if you were using a weak cipher”, and add a sentence saying that if you were using a ciphersuite without encryption then `MBEDTLS_CIPHER_NULL_CIPHER` alone is enough.

Reply:
I've amended the last commit.